docs/htmleditor-processing.adoc

```diff
@@ -0,0 +1,221 @@
+= HTML Editor processing
+
+image:images/v-500.svg[Since version,opts=inline] The default Guillotine HTML editor type provides macros and images processing in the `markup` field of the link:api.doc#HtmlEditorResult[HtmlEditorResult] GraphQL object type.
```

Comment:
```suggestion
image:images/v-500.svg[Since version,opts=inline] The default Guillotine HTML editor type provides macros and image source URLs processing in the `markup` field of the link:api.doc#HtmlEditorResult[HtmlEditorResult] GraphQL object type.
```

IMO image processing is too congfusing

Reply:
done